dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
```

Comment:
Could probably do `ConditionDirectoryNotEmpty=/sysroot/etc/ostree/kargs.d` here. systemd checks the condition at the time the unit would be started.

Reply:
I was thinking about that and I'm not sure. We may want more complicated checks in the future that the systemd conditionals can't support. That being said, I suppose it's fine for now; we can always change it to something more complex if it comes to that.

Reply:
Checking `kargs.d` inside `ignition-apply-kargs.sh` for now, might change to `ConditionDirectoryNotEmpty=/sysroot/etc/ostree/kargs.d` after testing.